main.js

```diff
@@ -35,13 +33,29 @@ function copyLink (el, isBanner) {
 
 }
 
+const shareContainer = document.querySelector('.invite-colleagues__share');
+
+const shouldIinitOshare = function () {
+	if (shareContainer.querySelector('[data-o-share--js]')) {
+		return false;
+	} else {
+		return true;
+	}
```

Comment:
You can simplify this to: `return shareContainer.querySelector('[data-o-share--js]')`

Reply:
oooooo nice!